Sources/NIOHTTP1/HTTPTypes.swift

```diff
@@ -80,6 +80,12 @@ public struct HTTPListHeaderIterator: Sequence, IteratorProtocol {
         return self
     }
     
+    @inline(__always)
+    public init(headerName: String,
+                headers: HTTPHeaders) {
+        self.init(headerName: headerName.utf8, headers: headers)
+    }
+    
     public init(headerName: String.UTF8View,
```

Comment:
We only need one `init`, not two, and this does not need to be `@inline(__always)`.